src/app/components/WorkDetail/WorkDetail.jsx

```diff
@@ -85,14 +79,17 @@ class WorkDetail extends React.Component {
           </div>
           <div className="nypl-row">
             <div className="nypl-column-full">
-              <h2>Work Detail</h2>
```

Comment:
Will there no longer be any h2?

Reply:
This label as-is was just a placeholder until the designs were provided. The design shows the title as a header [H1] instead and subtitle as [H2], I believe. It's unclear at this point since the design document and UI/UX documentation is out of sync.